pkg/vulnsrc/osv/osv.go

```diff
@@ -82,6 +82,11 @@ func (vs VulnSrc) Update(dir string) error {
 func (vs VulnSrc) save(eco ecosystem, entries []Entry) error {
 	err := vs.dbc.BatchUpdate(func(tx *bolt.Tx) error {
 		for _, entry := range entries {
+			//GHSA-IDs are already stored via ghsa package.
```

Comment:
It should be done when decoding to JSON.

Reply:
Moved